app/content/models/registration.py

```diff
@@ -51,30 +52,34 @@ class Meta:
 
     @classmethod
     def has_retrieve_permission(cls, request):
-        return check_has_access(cls.has_retrieve_access, request,)
+        return True
```

Comment:
Hvis denne er satt til `True` vil vel alle kunne se andres påmeldinger? Brukes denne eller `has_object_retrieve_permission()` for å hente ut én?

Reply:
Nei, permission sjekkes allikevel på `has_object_retrieve_permission` selv om denne er True

Reply:
Hva er hensikten med denne da?